test/conformance/serial/ptp.go

```diff
@@ -884,3 +1107,55 @@ func testCaseEnabled(testCase TestCase) bool {
 	}
 	return false
 }
+
+func processRunning(input string) map[string]bool {
+	// Regular expression pattern
+	processStatusPattern := `openshift_ptp_process_status\{config="([^"]+)",node="([^"]+)",process="([^"]+)"\} (\d+)`
+
+	// Compile the regular expression
+	processStatusRe := regexp.MustCompile(processStatusPattern)
+
+	// Find matches
+	processRunning := map[string]bool{"phc2sys": false, "ptp4l": false, "ts2phc": false, "gpspipe": false, "gpsd": false}
+
+	scanner := bufio.NewScanner(strings.NewReader(input))
+	for scanner.Scan() {
+		line := scanner.Text()
+		if matches := processStatusRe.FindStringSubmatch(line); matches != nil {
+			if _, ok := processRunning[matches[3]]; ok && matches[4] == "1" {
+				processRunning[matches[3]] = true
+			}
+
+		}
+	}
+	if err := scanner.Err(); err != nil {
+		fmt.Println("Error reading input:", err)
+	}
+	return processRunning
+}
+
+func clockStateByProcesses(input string) map[string]bool {
+	// Regular expression pattern
+	clockStatePattern := `openshift_ptp_clock_state\{iface="([^"]+)",node="([^"]+)",process="([^"]+)"\} (\d+)`
+
+	// Compile the regular expression
+	processStatusRe := regexp.MustCompile(clockStatePattern)
+
+	// Find matches
+	processClockState := map[string]bool{"phc2sys": false, "GM": false, "dpll": false, "ts2phc": false, "gnss": false}
+
+	scanner := bufio.NewScanner(strings.NewReader(input))
+	for scanner.Scan() {
+		line := scanner.Text()
+		if matches := processStatusRe.FindStringSubmatch(line); matches != nil {
+			if _, ok := processClockState[matches[3]]; ok && matches[4] == "1" {
+				processClockState[matches[3]] = true
```

Comment:
add break

Reply:
I have more than one entry in the map above so cant add a break here..